core/src/main/java/org/p2p/core/model/TitleValue.kt

```diff
@@ -0,0 +1,10 @@
+package org.p2p.core.model
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+class TitleValue(
```

Comment:
TransactionProgressHandler.kt is like a mapper itselfe it will be not good to override TransactionProgressHandler for needed stransaction types + some mappers
i mean it is base implementation
mappers better to add in overriden one if needed